### DISCUSSION TITLE
Why background shade colors are different between Input and Select?

### DISCUSSION THREAD

**Discussion by MariuzM** [heroui-inc/heroui#3944] [Feedback]
Looks like select is more darker than input when both set to primary, just wandering why is that? would be really nice to match them 
And when you hover them same time then its opposite, feels really confusing to view 

<img width="737" alt="image" src="https://github.com/user-attachments/assets/64568d60-832d-40c9-86cf-3f2e8b8343c6">

**Comment by wingkwong** [comment]
i think it's handling here: https://github.com/nextui-org/nextui/pull/3881

**Reply by MariuzM** [reply]
thank you, i hope this gets fixed soon so annoying, I'm trying to add my own color but really difficult for some reason to target that background with tailwind, maybe I'm doing it wrong

**Reply by MariuzM** [reply]
Also do add for my case I'm on light mode but this is how Input vs Select look 
<img width="689" alt="image" src="https://github.com/user-attachments/assets/bee05f6d-f6cd-411b-ac9e-7375d243dd9c">

**Reply by MariuzM** [reply]
For now done it like so
```
className="[&_button]:bg-primary-50 hover:[&_button]:bg-primary-100"
```
